docs/getting-started/android/setup.md

```diff
@@ -1,7 +1,55 @@
-# Getting started with fastlane for Android
+# Getting started with _fastlane_ for Android
 
-To be written soon, this will contain
+## Installing _fastlane_
 
-- Installing fastlane
-- Initial setup of fastlane for your projet
-- Where to go from there
+```no-highlight
+sudo gem install fastlane --verbose
+```
+
+## Setting up _fastlane_
+
+Navigate your terminal to your project's directory and run
+
+```no-highlight
+fastlane init
+```
+
+You'll be asked to confirm that you're ready to begin, and then for a few pieces of information. To get started quickly:
+
+1. Provide the package name for your application when asked (e.g. io.fabric.yourapp)
+1. Press enter when asked for the path to your json secret file
+1. Answer 'n' when asked if you plan on uploading info to Google Play via fastlane (we can set this up later)
```

Comment:
Why do we recommend `n`? Shouldn't it be something like: if your app is already on Google Play, press `y`, otherwise `n`, or similar?